compiler-core/Cargo.toml

```diff
@@ -72,6 +72,8 @@ bincode = "1.3.3"
 globset = { version = "0.4.9", features = ["serde1"] }
 # Compact and cheap to clone immutable string type
 smol_str = { version = "0.1", features = ["serde"] }
+# Checksums
+sha2 = "0.9.8"
```

Comment:
Let's use xxhash, which is much faster than sha256.

https://crates.io/crates/xxhash-rust